rust/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+# We name the package `synapse` so that things like logging have the right
+# logging target.
+name = "synapse"
+version = "0.1.0"
```

Comment:
Should this match the version specified in pyproject.toml?

Reply:
I don't believe it has any affect, its just a mandatory field in Rust. TBH I would rather not try to keep the two in sync.

Reply:
I suppose we're not planning on publishing this to crates.io or any other registry.

Maybe drop a quick comment e.g.

```toml
# dummy version. See pyproject.toml for the Synapse's version number.
```

> TBH I would rather not try to keep the two in sync.

Agreed.